climada/engine/impact.py

```diff
@@ -1091,11 +1091,8 @@ def _selected_exposures_idx(self, coord_exp):
         if coord_exp is None:
             sel_exp = []
         else:
-            sel_exp =  [
-                j
-                for j, coord in enumerate(self.coord_exp)
-                if coord in coord_exp
-                ]
+            assigned_idx = u_coord.assign_coordinates(self.coord_exp, coord_exp, threshold=0)
+            sel_exp = list((assigned_idx >= 0).nonzero()[0])
```

Comment:
Is it really necessary to convert to a list? Keeping a numpy array would not work?

Reply:
I meant to stick to the previous behavior, but of course you are right. Fixed in latest commit.